build.gradle

```diff
@@ -37,7 +37,6 @@ plugins {
     alias(libs.plugins.versions)
     alias(libs.plugins.spotbugs)
     alias(libs.plugins.nexus)
-    alias(libs.plugins.artifactory)
```

Comment:
Can this be removed from `libs.versions.toml` now?

Reply:
Oh whoops. Definitely meant to remove it the first time. Yeah, it's unreferenced now. Removed